MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+# Files to be included in the distribution
```

Comment:
FWIW, you can also use the manifest to exclude files (as well as perform other actions).